cli/commands/flags.go

```diff
@@ -172,6 +172,9 @@ const (
 	TerragruntProviderCacheRegistryNamesFlagName = "terragrunt-provider-cache-registry-names"
 	TerragruntProviderCacheRegistryNamesEnvName  = "TERRAGRUNT_PROVIDER_CACHE_REGISTRY_NAMES"
 
+	TerragruntFeatureMapFlagName = "feature"
+	TerragruntFeatureMapEnvName  = "TERRAGRUNT_FLAG"
```

Comment:
Sorry, should we actually call it `TERRAGRUNT_FEATURE` instead?

Reply:
Sure, will update